libs/blocks/global-navigation/global-navigation.js

```diff
@@ -1025,27 +1025,32 @@ const getSource = async () => {
 };
 
 export default async function init(block) {
-  try {
-    const { mep } = getConfig();
-    const sourceUrl = await getSource();
-    const [url, hash = ''] = sourceUrl.split('#');
-    if (hash === '_noActiveItem') {
-      setDisableAEDState();
-    }
-    const content = await fetchAndProcessPlainHtml({ url });
-    if (!content) return null;
-    const gnav = new Gnav({
-      content,
-      block,
+  const { mep } = getConfig();
+  const sourceUrl = await getSource();
+  const [url, hash = ''] = sourceUrl.split('#');
+  if (hash === '_noActiveItem') {
+    setDisableAEDState();
+  }
+  const content = await fetchAndProcessPlainHtml({ url });
+  if (!content) {
+    const error = new Error('Could not create global navigation. Content not found!');
+    error.tags = 'errorType=error,module=gnav';
+    error.url = url;
+    window.lana.log(`${error.message} | gnav-source: ${error.url} | href: ${window.location.href}`, {
```

Comment:
Any reason for not using `lanaLog` ?

Reply:
This was an oversight, fixing it now. Thanks for pointing it out.